### PR TITLE
Added connection_setting_idletimeout to elb_lb module

### PR DESCRIPTION
**Pull request by lluishernandez**
This PR aim to bring the capability to set the `connecting_settings.timeout` value using ansible. All the changes are contained exclusively into `cloud.amazon.ec2_elb_lb` module.

The only missing bit is at the documentation part, it's necessary to define the version this PR will included (or not).

### CODE REVIEW

File: cloud/amazon/ec2_elb_lb.py [hunk 1]
```diff
@@ -120,6 +120,13 @@
       - An associative array of stickness policy settings. Policy will be applied to all listeners ( see example )
     required: false
     version_added: "2.0"
+  connection_settings_idletimeout:
+    description:
+      - Wait an specified timeout keeping open idle connections.
+    required: false
+    aliases: []
+    default: 60
+    version: ??
```
Comment (lluishernandez) [inline cloud/amazon/ec2_elb_lb.py:129]: Will be necessary to set the correct `version` number here, before merge.

File: cloud/amazon/ec2_elb_lb.py [hunk 2]
```diff
@@ -900,6 +921,11 @@ def main():
     if connection_draining_timeout and not elb_man._check_attribute_support('connection_draining'):
         module.fail_json(msg="You must install boto >= 2.28.0 to use the connection_draining_timeout attribute")
 
+    if connection_settings_idletimeout and not self._check_attribute_support('connecting_settings'):
```
Comment (marvinpinto) [inline cloud/amazon/ec2_elb_lb.py]: Should this be `elb_man._check_attribute_support('connection_settings')` instead?

**Comment by marvinpinto** [comment]
Was just looking at the `get_info` method, should there be an entry for `connection_settings_idletimeout` there similar to https://github.com/lluishernandez/ansible-modules-core/blob/add_connection_idletimeout_to_elb_lb/cloud/amazon/ec2_elb_lb.py#L457-L458 ?

**Comment by marvinpinto** [comment]
Might possibly also need a `connection_settings_idletimeout` entry in the `argument_spec` dict? https://github.com/lluishernandez/ansible-modules-core/blob/add_connection_idletimeout_to_elb_lb/cloud/amazon/ec2_elb_lb.py#L841-L856

**Comment by lluishernandez** [comment]
Hello! I've been trying to execute this PR, but I'm getting the following error:

```
failed: [localhost] => {"failed": true}
msg: unsupported parameter for module: connection_settings_idletimeout

FATAL: all hosts have already failed -- aborting
```

I suppose there is another module that validate the parameters used on playbooks, is there any work around to let me check this PR works properly?? Did I missed anything?

**Comment by gregdek** [comment]
@lluishernandez Thanks for submitting this pull request. Unfortunately, it appears that the underlying code has changed since your submission, and this PR now needs to be rebased. 

If you believe that this PR still applies, please rebase and we will put it into our community review process as soon as possible. 

Once you have rebased this PR, please add a comment with the text "ready_for_review" and we will continue with the review process.  Thanks!

**Comment by gregdek** [comment]
@lluishernandez A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes.

**Comment by robynbergeron** [comment]
@lluishernandez Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If we don't hear from you within another 14 days, we will close this pull request.

**Comment by gregdek** [comment]
Closed due to inactivity by submitter.

**Pull request closed without merging**